src/dns-lookup/templates/app.vue

```diff
@@ -149,7 +154,7 @@ limitations under the License.
 
                     this.setRegistrar(text)
 
-                    if (!this.$data.linked) window.history.pushState({}, "", `?domain=${encodeURIComponent(text)}`)
+                    if (!this.$data.linked) window.history.replaceState({}, "", `?domain=${encodeURIComponent(text)}`)
```

Comment:
This no longer inserts a new history item for each search, which defeats the entire point of adding support for the back button?